skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkikoUIView.kt

```diff
@@ -181,13 +181,32 @@ class SkikoUIView : UIView, UIKeyInputProtocol, UITextInputProtocol,
         return _pointInside(skiaPoint, withEvent)
     }
 
+    /**
```

Comment:
Use /* comments for private things

Reply:
It has a side effect of property description not being shown on hover in IDE.

Reply:
I guess, it's a question to IDE developers, fixed.